deploy/kessel-relations-deploy.yaml

```diff
@@ -0,0 +1,81 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: relations
+objects:
+  - apiVersion: authzed.com/v1alpha1
+    kind: SpiceDBCluster
+    metadata:
+      name: ${CLOWDAPP_NAME}-spicedb
+    spec:
+      config:
+        logLevel: debug
+        replicas: 1
+        datastoreEngine: postgres
+        datastoreBootstrapFiles: /etc/bootstrap/schema.yaml
+      secretName: ${ENV_NAME}-spicedb-config
```

Comment:
It ended up just being called spicedb-config - no need for the ENV_NAME here after all